tests/test_other.py

```diff
@@ -9655,29 +9654,15 @@ def test_no_excessive_invoke_functions_are_generated_when_exceptions_are_enabled
       self.assertNotContained('invoke_ii', output)
       self.assertNotContained('invoke_v', output)
 
-  def test_add_emscripten_metadata(self):
+  def test_emscripten_metadata(self):
+    run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world.c')])
+    self.assertNotIn(b'emscripten_metadata', open('a.out.wasm', 'rb').read())
```

Comment:
might be slightly nicer to save that `b'emscripten_metadata'` to a local, to avoid the risk of typos. but lgtm either way.